### ISSUE TITLE
UDS corpus encoding error

### ISSUE THREAD

**Issue by Chroner2** [decompositional-semantics-initiative/decomp#3]
Hello,
I am trying to go through the tutorial but it seems I get stuck on the first couple of lines. When I try to load the UDS corpus (i.e.` uds = UDSCorpus()`) I get a JSON encoding error: `json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16335364 (char 16335363)`

**Comment by sidsvash26** [comment]
Yeah, I also get a similar error when downloading v0.1.3:

I installed v0.1.3 as follows:
```shell
conda create -n py3.8 python=3.8
source activate py3.8 
conda install pip
pip install --user -e git://github.com/decompositional-semantics-initiative/decomp.git@v0.1.3#egg=decomp
```
When I import UDS in Python as follows:
```python
from decomp import UDSCorpus
uds = UDSCorpus()
```

I get the following error:
```python
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/sidvash/src/decomp/decomp/semantics/uds.py", line 88, in __init__
    corp_split = self.__class__.from_json(fpath)
  File "/Users/sidvash/src/decomp/decomp/semantics/uds.py", line 183, in from_json
    graphs_json = json.load(infile)
  File "/anaconda3/envs/py3.8/lib/python3.8/json/__init__.py", line 293, in load
    return loads(fp.read(),
  File "/anaconda3/envs/py3.8/lib/python3.8/json/__init__.py", line 357, in loads
    return _default_decoder.decode(s)
  File "/anaconda3/envs/py3.8/lib/python3.8/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/anaconda3/envs/py3.8/lib/python3.8/json/decoder.py", line 353, in raw_decode
    obj, end = self.scan_once(s, idx)
json.decoder.JSONDecodeError: Expecting property name enclosed in double quotes: line 1 column 167268161 (char 167268160)

```

**Comment by aaronstevenwhite** [comment]
Does this error replicate on the supplied Docker container?

**Comment by Chroner2** [comment]
I will try and let you know, once I cool off with my schedule.

**Comment by aaronstevenwhite** [comment]
I believe this issue has been fixed by the same fix to PredPatt mentioned in #5.
